aa_stripe/models.py

```diff
@@ -125,6 +125,52 @@ def create_at_stripe(self):
         self.save()
         return self
 
+    def update_at_stripe(self, stripe_token, set_default):
+        is_default = self.customer.default_card.pk is self.pk
+        set_default = is_default if set_default is None else set_default
+
+        if is_default and not set_default:
+            raise StripeLogicalError()
+
+        stripe.api_key = stripe_settings.API_KEY
+
+        customer = stripe.Customer.retrieve(self.customer.stripe_customer_id)
```

Comment:
if the card is default and it has a set default flag, why do you need to update the customer? it is default already, right?

Reply:
When the card that need to be updated is not he default card I need to change the default source on the customer so when i update the card with source attribute it is the one that is changed, because stripe api updates the default source(card) when we use source attribute on stripe customer object.